index.html

```diff
@@ -123,97 +117,75 @@
+### Extension to the <a>PerformanceResourceTiming</a> interface
 
-    If the `maxSize` parameter is less than the number of elements currently stored in the buffer, no elements in the buffer are to be removed and the user agent MUST NOT fire the `servertimingbufferfull` event.
+<pre class='idl'>
+[Exposed=(Window,Worker)]
+partial interface PerformanceResourceTiming {
+  readonly attribute FrozenArray&lt;PerformanceServerTiming&gt; serverTiming;
+};
+</pre>
 
-    The <var>maxSize</var> parameter sets the maximum number of <a>PerformanceServerTiming</a> resources that will be stored in the buffer.
-    </dd>
-
-    <dt><dfn>onservertimingbufferfull</dfn> attribute</dt>
-    <dd>The event handler for the `servertimingbufferfull` event. Immediately after the buffer used to store the list of <a>PerformanceServerTiming</a> resources becomes full, the user agent MUST fire a simple event named `servertimingbufferfull` that bubbles, isn't cancelable, has no default action, at the <a>`Performance`</a> object [[!NAVIGATION-TIMING-2]].</dd>
-  </dl>
-</section>
+<p>The <dfn>serverTiming</dfn> field returns a sequence of <a>PerformanceServerTiming</a> entries.</p>
 
 ## Process
 
 ### Processing Model
+<p>For each resource <a title='fetch' href="https://fetch.spec.whatwg.org/#concept-fetch">fetched</a> by the current <a href="https://www.w3.org/TR/html5/browsers.html#browsing-context">browsing context</a>, excluding resources fetched by cross-origin stylesheets fetched with <code>no-cors</code> policy, perform the following steps:</p>
+
 <ol>
-  <li>Once the <a data-cite="!HTML5#create-a-document-object">Window object of the current document is created</a> the user agent MUST create a <a data-cite="performance-timeline-2#performanceentrylist">`PerformanceEntryList`</a> object (<dfn>primary buffer</dfn>) to store the list of <a>PerformanceServerTiming</a> resources.</li>
-  <li>Set the <a>primary buffer</a> to a size of 150, unless otherwise specified by the user agent or set by the <a>setServerTimingBufferSize</a> method.</li>
+  <li>Let <dfn>entryList</dfn> be a new empty <a>Sequence</a>.
+  </li>
   <li>For each server-specified metric received via the <a>Server-Timing header field</a> perform the following steps:
-    <ol data-link-for="ServerEntry">
-      <li id="step-create-object">Create a new <a>PerformanceServerTiming</a> object and set <a>entryType</a> to the DOMString `server`.</li>
-      <li>Set the <a>name</a> to the <a data-cite="!HTML5#resolve-a-url">resolved URL</a> of the requested resource.</li>
+    <ol data-link-for="PerformanceServerTiming">
+      <li id="step-create-object">Let <dfn>entry</dfn> be a new <a>PerformanceServerTiming</a>.</li>
+      <li>Set <a>entryType</a> to the DOMString "`server`".</li>
+      <li>Set <a>name</a> to the <a data-cite="!HTML5#resolve-a-url">resolved URL</a> of the requested resource.</li>
```

Comment:
Maybe add a 'Parse the header value according to ABNF grammar specified in "The `Server-Timing` Header Field"'?